map/map.js

```diff
@@ -93,6 +93,18 @@ var DefineMap = Construct.extend("DefineMap",{
 					this.constructor.seal
 				);
 			});
+
+			var _computedGetter = Object.getOwnPropertyDescriptor(prototype, "_computed").get;
```

Comment:
is this going to be a problem in `define(someProptotoype)`-style usage?  Should these changes be made w/i `define()` directly then?

Reply:
No, `define(obj)` does not set up an observer for `can.keys`, that happens in DefineMap's `can.getOwnEnumerableKeys` symbol: https://github.com/canjs/can-define/blob/3fe59b046311d436ccc467d4a221241d6009297f/map/map.js#L228-L232